vmhost/hostCore/execution.go

```diff
@@ -79,6 +79,10 @@ func (host *vmHost) performCodeDeployment(input vmhost.CodeDeployInput) (*vmcomm
 
 	_, _, metering, output, runtime, _ := host.GetContexts()
 
+	if host.EnableEpochsHandler().IsRuntimeCodeSizeFixEnabled() {
```

Comment:
:+1: common method for both create & upgrade

Reply:
Moved into an even more general code path.